docs/source/usage/plugins/phaseSpace.rst

```diff
@@ -37,47 +37,61 @@ Option                                 Usage
 Output
 ^^^^^^
 
-Values are given as *charge density* per phase space bin.
-In order to scale to a simpler *charge of particles* per :math:`\mathrm{d}r_i` and :math:`\mathrm{d}p_i` -bin multiply by the cell volume:
+The histograms are stored in ``.hdf5`` files in the ``simOutput/...`` directory.
 
-.. code:: python
+...
 
-   ps = ... # get data set from h5py
+Values are given as *charge density* per phase space bin.
+In order to scale to a simpler *charge of particles* per :math:`\mathrm{d}r_i` and :math:`\mathrm{d}p_i` -bin multiply by the cell volume ``dV``.
 
-   # 3D3V example
-   dV = ps.attrs['dV'] * ps.attrs['dr_unit']**3
-   charge_per_bin = ps[:,:] * ps.attrs['sim_unit'] * dV
+Analysis
+^^^^^^^^
 
-Out-of-Range Behavior
-^^^^^^^^^^^^^^^^^^^^^
+The easiest way is to load the data in Python:
 
-Particles that are *not* in the range of ``<ValL>``/``<ValR>`` get automatically mapped to the lowest/highest bin respectively.
-Take care about that when setting your range and during analysis of the results.
+.. code:: python
 
-Spatial Offset
-^^^^^^^^^^^^^^
+   from picongpu.plugins.phase_space import PhaseSpace
+   import matplotlib.pyplot as plt
+   from matplotlib.colors import LogNorm
+   import numpy as np
 
-The spatial extent of the output might change due to a moving window.
-Additional attributes are provided to retrieve that spatial information:
 
-See this python example:
+   # load data
+   phase_space = PhaseSpace('/home/axel/runs/foil_001')
+   e_ps, e_ps_meta = phase_space.get('e', ps='ypy', iteration=1000)
 
-.. code:: python
+   # unit conversion from SI
+   mu = 1.e6  # meters to microns
+   e_mc_r = 3.6618e21  # electrons: kg * m / s to beta * gamma
 
-   ps = ... # get data set from h5py
+   # plotting
+   plt.imshow(
+       np.abs(e_ps) * e_ps_meta.dV,
```

Comment:
QUESTION:
Doesn't `e_ps` contain under- and over-flow bins as well?
Shoudn't they be removed from plotting, it the range is used as below?

Reply:
I had this in a draft but did not add it to the functionality yet.
A section below it is mentioned, that the highest and lowest p bin will also contain overflows.

Reply:
how about `np.abs(e_ps[:, 1:-1])`and a brief comment - otherwise it gets confusing with the detailed (and good) description of the under- and over-flow bins below.

Reply:
it's a bit more than that, removing two bins also changes the extent, etc. I just wanted to avoid overcomplicating it for the first round :)